tests/integration/pre-suite/01_websphere_module_install.rb

```diff
@@ -0,0 +1,33 @@
+test_name 'FM-3804 - C94728 - Plug-in Sync Module from Master with Prerequisites Satisfied on Agent'
```

Comment:
Test name is different from the test case name in TestRail. This is important in the situation where the TestRail test case ID changes for some reason. (Like when we re-organized TestRail a while back.)